docs/faq/gradient_compression.md

```diff
@@ -0,0 +1,107 @@
+# Gradient Compression
+
+Gradient Compression reduces communication bandwidth to make distributed training with GPUs more scalable and efficient without significant loss in convergence rate or accuracy.
```

Comment:
Its not restricted to distributed or GPUs

Reply:
Also there is no guarantee that you won't loss convergence or accuracy significantly.

Reply:
Thanks, but the info provided yesterday was limited to this particular scenario. Should you or another subject matter expert like to provide more context at a later time, it can be included, likely at a later time. In the meantime, I'll add some vague caveats to deal with the apparent lack of other scenarios and information.